README.md

```diff
@@ -1,83 +1,135 @@
+1. Rename the placeholder config file [config-placeholder.json](config-placeholder.json) to config.json
+1. Update config.json with the credentials for the enpoint(s) you want to send to. See [Configure endpoints and authentication](#configure-endpoints-and-authentication) below for additional details
+1. Run program.py
 
-To test the sample after running it:
+## To Test this Sample:
 
-1. Run `python test.py`
+### Option 1
+1. Run test.py
 
-or
+### Option 2
 
 1. Install pytest `pip install pytest`
 1. Run `pytest program.py`
 
-## Configure constants for connecting and authentication
-
-The sample is configured using the file [config.placeholder.ini](config.placeholder.ini). Before editing, rename this file to `config.ini`. This repository's `.gitignore` rules should prevent the file from ever being checked in to any fork or branch, to ensure credentials are not compromised.
-
-The SDS Service is secured by obtaining tokens from Azure Active Directory. Such clients provide a client application identifier and an associated secret (or key) that are authenticated against the directory. You must replace the placeholders in your `config.ini` file with the authentication-related values you received from OSIsoft.
-
-The values to be replaced are in `config.ini`:
-
-```ini
-[Configurations]
-Namespace = Samples
-
-[Access]
-Resource = https://dat-b.osisoft.com
-Tenant = REPLACE_WITH_TENANT_ID
-ApiVersion = v1
-
-[Credentials]
-ProducerToken = REPLACE_WITH_TOKEN_STRING
-ClientId = REPLACE_WITH_APPLICATION_IDENTIFIER
-ClientSecret = REPLACE_WITH_APPLICATION_SECRET
+## Customizing the Application
+This application can be customized to send your own custom types, containers, and data by modifying the [OMF-Types.json](OMF-Types.json), 
+[OMF-Containers.json](OMF-Containers.json), and [OMF-Data.json](OMF-Data.json) files respectively. Each one of these files contains an array of OMF json objects, which are
+created in the endpoints specified in [config.json](config-placeholder.json) when the application is run. For more information on forming OMF messages, please refer to our 
+[OMF version 1.1 documentation](https://omf-docs.osisoft.com/documentation_v11/Whats_New.html).  
+  
+In addition to modifying the json files mentioned above, the get_data function in [program.py](program.py) should be updated to populate the OMF data messages specified in 
+[OMF-Data.json](OMF-Data.json) with data from your data source.  
+  
+Finally, if there are any other activities that you would like to be running continuously, this logic can be added under the while loop in the main() function of 
+[program.py](program.py).
+
+## Configure Endpoints and Authentication
+
+The sample is configured using the file [config-placeholder.json](config-placeholder.json). Before editing, rename this file to `config.json`. This repository's `.gitignore` rules should prevent the file from ever being checked in to any fork or branch, to ensure credentials are not compromised.
```

Comment:
@dcbroad3 I mentioned it in both the "To Run this Sample" section and the "Customizing the Application" sections in case the reader is skipping around the document. If it is too much I can remove it.

Reply:
Generally I'd only say it once, but if you're concerned they'll skip it in one section, I'm sure it's fine to leave it.